backend/apps/blog/views.py

```diff
@@ -2,34 +2,19 @@
 from rest_framework import status
 from rest_framework.response import Response
 from rest_framework.views import APIView
-from rest_framework.decorators import api_view
 from .models import *
-from .serializer import *
+from .serializers import *
 
 
 # Type Hint
 import typing as t
 RedirectOrResponse = t.Union[HttpResponseRedirect, HttpResponse]
 
-descriptions = [
-    {
-        "Welcome": "Blog Api v1",
-        "Api Version 1.0": "/docs or /redoc",
-    }
-]
-
-@api_view(["GET"])
-def intro(request: HttpRequest) -> Response:
-    return Response(descriptions, status=status.HTTP_200_OK)
 
 #Blog All Start
 class Get_blog_all(APIView):
```

Comment:
Should be renamed to Get_all_blog for better readability

Reply:
i will be fix.